templates/page.tpl.php

```diff
@@ -73,6 +73,7 @@
  * @see template_process()
  * @see html.tpl.php
  */
+$form_below_main = $order_enabled && $order === 'below-main';
```

Comment:
Does it have to be that complicated? I’d prefer a simple Boolean, e.g. `$reverse = TRUE` – or do you expect more kinds of layout variations? :flushed: 

If really needed, additional page variables should be set in the page preprocess function, not the template file (we've tried that, it wasn’t such a good idea). The code should also take into account that the variable might *not* be set, e.g. `campaignion_layout` does not have to be enabled.

Related to https://github.com/moreonion/campaignion/pull/273#discussion_r609370054

Reply:
Haha, having the same discussion for `campaignion_layout`... I think it's not overly complicated. If we are confident enough, that we are happy with a boolean, fine with me to change it into a boolean. I do not expect more variations, but I would not rule out the possibility. That's why I chose a string.

Reply:
But granted, `$reverse = TRUE` does sound much simpler... :innocent:

Reply:
Looks a lot clearer now, thanks!

Still, without `campaignion_layout` you’ll get a "Undefined variable" notice → I’d suggest to use `empty()` in the if conditions.

Reply:
Changed.